index.ts

```diff
@@ -0,0 +1,43 @@
+const express = require('express');
+const app = express();
+const port = 3000;
+const todo = require("./src/todo");
+const MyTodo = new todo.Todo();
+
+app.use(express.json()) // for parsing application/json
+app.use(express.urlencoded({ extended: true })) // for parsing application/x-www-form-urlencoded
+
+app.get('/', function (req, res) {
+    const listDes = [];
+    if (MyTodo.list.length !== 0) {
+        const len = MyTodo.list.length;
+        for (let i = 0; i < len; i++) {
+            listDes.push(MyTodo.list[i].description)
+        }
+        res.send(listDes + '\n');
```

Comment:
Per the updates by Emmanuel we are looking for a JSON response.  look at the `.json` method in express response objects and make use of that accordingly.